### ISSUE TITLE
Improve the Readme file.

### ISSUE THREAD

**Issue by sandptel** [Sandeep-1507/captcha_iitkgp#1]
Change the old readme file to a better visual.

**Comment by Veerhan-glitch** [comment]
hey can u assign it to me?
(see [ALG](https://github.com/veerhan-glitch/alg) for a reference of my work)

**Comment by sandptel** [comment]
> hey can u assign it to me?
> (see [ALG](https://github.com/veerhan-glitch/alg) for a reference of my work)

Ok sure.

**Comment by Veerhan-glitch** [comment]
#  CRNN OCR 
[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1Fa_WgMyz9ZrWhPtcJ1DGR5GiQuix_1wM?usp=sharing) [![Kaggle Dataset](https://img.shields.io/badge/Kaggle-Dataset-blue.svg)](https://www.kaggle.com/datasets/sandeep1507/captchaimgdata)

This repository provides you with a deployable model to solve captchas which contains characters with random alphanumeric character which has been strikethrough vertically and/or horizontally. 

The tool to scrap these images from the website is available on [google colab](https://colab.research.google.com/drive/1cjwTIkQuU0ZUtSY8QKtg69gHR5PLnKLm?usp=sharing)

Check out my [LinkedIn post](https://www.linkedin.com/posts/sandppatel_i-have-created-a-crnn-model-to-solve-the-activity-7087672345380683777-3DOk?utm_source=share&utm_medium=member_desktop) for the description, use case, and other relevant information.

## Table of Contents

- [Type of solvable captchas:](#type-of-solvable-captchas:)
- [How to use](#how-to-use)
- [Accuracy](#accuracy)

## Type of solvable captchas:

![Image Alt](Data/testset/2CJ763.jpeg)

## How to use
Make sure you have a Google account and are signed in to access and run the notebook.
1. Go to [Colab Notebook](https://colab.research.google.com/drive/1mafV7t0o0mPMyH1rObs60o3DNy8KDsUs?usp=sharing)

2. Add your folder path to [captchasolve](https://colab.research.google.com/drive/1mafV7t0o0mPMyH1rObs60o3DNy8KDsUs#scrollTo=gA_ikL1Z5tQv&line=4&uniqifier=1) function and get a list of strings with predicted texts.

3. Follow the instructions provided in the notebook to run and interact with the project.

## Accuracy

[![Kaggle Dataset](https://img.shields.io/badge/Kaggle-Dataset-blue.svg)](https://www.kaggle.com/code/sandeep1507/testing-model-accuracy) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1mafV7t0o0mPMyH1rObs60o3DNy8KDsUs?usp=sharing)

1. Test the accuracy of model using the notebooks given above on testdataset.

add it in the first line

**Comment by Veerhan-glitch** [comment]
above is what i have done till no, i am not making a pr as i am not done yet as i needed the code to run for a better understanding which is not happening,

it is giving me some errors so can u pls help me debug them?

in 
from tensorflow.keras.models import Sequential, save_model, load_model
model = load_model("/content/kaggle/working", compile = True)

it gives /content/kaggle/working is not found


(also should i also do issue #8 as i am already doing it to run the code)

**Comment by sandptel** [comment]
/content/kaggle/working is the part where it fetches the dataset from the kaggle server. You either run the notebook already provided in kaggle or simply change the location to " /content/kaggle/working " to the location of the downloaded dataset on your server or pc.
